src/modules/rest/api/src/main/java/it/geosolutions/geostore/services/rest/IdPLoginRest.java

```diff
@@ -43,14 +45,24 @@ public interface IdPLoginRest {
 
     @GET
     @Path("/{provider}/login")
-    @Secured({"ROLE_USER", "ROLE_ADMIN", "ROLE_ANONYMOUS"})
+    @Secured({"ROLE_USER", "ROLE_ADMIN", "ROLE_GUEST", "ROLE_ANONYMOUS"})
```

Comment:
We didn't support this for the moment, so please remove
```suggestion
    @Secured({"ROLE_USER", "ROLE_ADMIN", "ROLE_ANONYMOUS"})
```